rbd/admin/groupsschedule.go

```diff
@@ -0,0 +1,42 @@
+//go:build !nautilus
+// +build !nautilus
```

Comment:
can we remove this build tag as its not required and still not support in upstream release in ceph?

Reply:
Done